data/brands/shop/clothes.json

```diff
@@ -3602,6 +3602,7 @@
       "locationSet": {"include": ["ru"]},
       "tags": {
         "brand": "Familia",
+        "brand:wikidata": "Q128004144",
```

Comment:
Let's switch to https://www.wikidata.org/wiki/Q127514809, Q128004144 is likely a duplicate. I can merge them.

Reply:
Yes it is. I should've to search better. Thank you.